FastSimulation/Calorimetry/src/CalorimetryManager.cc

```diff
@@ -180,71 +181,82 @@ void CalorimetryManager::reconstruct(RandomEngineAndDistribution const* random)
       if(debug_)
 	mySimEvent->print();
     }
+
+  initialize();
+  
+  LogInfo("FastCalorimetry") << "Reconstructing " << (int) mySimEvent->nTracks() << " tracks." << std::endl;
+  for( int fsimi=0; fsimi < (int) mySimEvent->nTracks() ; ++fsimi) {
+    
+    FSimTrack& myTrack = mySimEvent->track(fsimi);
+    
+    reconstructTrack(myTrack, random);
+  } // particle loop
+  //  LogInfo("FastCalorimetry") << " Number of  hits (barrel)" << EBMapping_.size() << std::endl;
```

Comment:
Is this commented out code needed? If not, you should better remove (in particular L196, which looks like a remnant of some private debugging)